docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -40,67 +40,58 @@ ascending order.
-metric to use, and ``smaller_is_better``, which defines whether to sort the checkpoints in ascending
-or descending order with respect to the specified metric.
+metric to use, and ``order_by``, which defines whether to sort the checkpoints in ascending or
+descending order with respect to the specified metric.
 
 .. code:: python
 
-   from determined.experimental import client
+   from determined.experimental import checkpoint, client
 
    checkpoint = (
-       client.get_experiment(id).top_checkpoint(sort_by="accuracy", smaller_is_better=False)
+       client.get_experiment(id).list_checkpoints(sort_by="accuracy", order_by=checkpoint.CheckpointOrderBy.DESC)
    )
 
-You may also query multiple checkpoints at the same time using the
-:meth:`~determined.experimental.client.Experiment.top_n_checkpoints` method. Only the single best
-checkpoint from each trial is considered; out of those, the checkpoints with the best validation
-metric values are returned in sorted order, with the best one first. For example, the following
-snippet returns the top five checkpoints from distinct trials of a specified experiment.
+You may also query checkpoints by preset checkpoint parameters, as defined by
+:class:`~determined.experimental.checkpoint.CheckpointSortBy`. The following code snippet fetches
+all checkpoints for the experiment sorted by the checkpoint's trial ID in descending order.
 
 .. code:: python
 
-   from determined.experimental import client
+   from determined.experimental import checkpoint, client
 
-   checkpoints = client.get_experiment(id).top_n_checkpoints(5)
-
-This method also accepts ``sort_by`` and ``smaller_is_better`` arguments.
+   checkpoints = client.get_experiment(id).list_checkpoints(
+      sort_by=checkpoint.CheckpointSortBy.TRIAL_ID,
+      order_by=checkpoint.CheckpointOrderBy.DESC
+   )
 
 :class:`~determined.experimental.client.Trial` is used for fine-grained control over checkpoint
-selection within a trial. It contains a :meth:`~determined.experimental.client.Trial.top_checkpoint`
-method, which mirrors :meth:`~determined.experimental.client.Experiment.top_checkpoint` for an
-experiment. It also contains :meth:`~determined.experimental.client.Trial.select_checkpoint`, which
-offers three ways to query checkpoints:
+selection within a trial. It contains a
+:meth:`~determined.experimental.client.Trial.list_checkpoints` method, which mirrors
+:meth:`~determined.experimental.client.Experiment.list_checkpoints` for an experiment.
 
-#. ``best``: Returns the best checkpoint based on validation metrics as discussed above. When using
-   ``best``, ``smaller_is_better`` and ``sort_by`` are also accepted.
-#. ``latest``: Returns the most recent checkpoint for the trial.
-#. ``uuid``: Returns the checkpoint with the specified UUID.
-
-The following snippet showcases how to use the different modes for selecting checkpoints.
+The following snippet showcases how to select the latest checkpoint for a trial.
```

Comment:
The snippet below demonstrates how to select the most recent checkpoint for a trial.

- or -
The snippet below demonstrates how to select the latest checkpoint for a trial.